app/src/main/java/com/jvoye/tasky/agenda/presentation/components/AgendaItemCard.kt

```diff
@@ -35,27 +44,35 @@ import com.jvoye.tasky.core.presentation.designsystem.theme.surfaceHigher
 
 @Composable
 fun AgendaItemCard(
+    modifier: Modifier = Modifier,
     agendaItem: AgendaItem,
+    onAgendaItemClick: (agendaItemId: Int) -> Unit,
+    onAgendaItemMenuClick: (agendaItemId: Int) -> Unit,
     action: (AgendaAction) -> Unit,
-    onAgendaItemClick: () -> Unit    
 ) {
+    val menuItems = listOf(
+        AgendaMenuType.OPEN to stringResource(R.string.open),
+        AgendaMenuType.EDIT to stringResource(R.string.edit),
```

Comment:
You might want to take the same approach here with `AgendaMenuType` like the FAB entries. Instead of keeping a hardcoded `listOf(type to stringResource(...))` in the composable, you can let each enum map itself to a `UiText` 😁.

```kotlin
fun AgendaMenuType.toUiText(): UiText = when (this) {
    AgendaMenuType.OPEN   -> UiText.StringResource(R.string.open)
    AgendaMenuType.EDIT   -> UiText.StringResource(R.string.edit)
    AgendaMenuType.DELETE -> UiText.StringResource(R.string.delete)
}
```